insights/v1alpha1/doc.go

```diff
@@ -0,0 +1,8 @@
+// +k8s:deepcopy-gen=package,register
+// +k8s:defaulter-gen=TypeMeta
+// +k8s:openapi-gen=true
+
+// +kubebuilder:validation:Optional
```

Comment:
What's the motivation behind making fields optional by default?

Reply:
Hmm no real motivation....I guess just copy/paste from other `doc.go` files...